test/e2e/tests/settings/account-token-list.spec.ts

```diff
@@ -41,26 +42,15 @@ describe('Settings', function () {
     await withFixtures(
       {
         fixtures: new FixtureBuilder().withConversionRateDisabled().build(),
-        title: this.test.fullTitle(),
+        title: this.test?.fullTitle(),
       },
       async ({ driver }) => {
-        await logInWithBalanceValidation(driver);
-
-        await driver.clickElement(
-          '[data-testid="account-overview__asset-tab"]',
+        await loginWithBalanceValidation(driver);
+        await new AssetListPage(driver).check_tokenAmountIsDisplayed('25 ETH');
```

Comment:
mmm I think we could omit this, despite being in the original test, basically because we already check for  eth balance in the default value for the loginWithBalanceValidation

what do you think?

Reply:
oh I see! Thank you for clarifying :pray: makes sense!!